doc/source/examples/cfd-dem/square-fluidized-bed/square-fluidized-bed.rst

```diff
@@ -0,0 +1,392 @@
+==================================
+Gas-Solid Fluidized Bed
+==================================
+
+It is strongly recommended to visit `CFD-DEM parameters <../../../parameters/unresolved_cfd-dem/unresolved_cfd-dem.html>`_ for more detailed information on the concepts and physical meaning of the parameters CFD-DEM.
+
+Features
+----------------------------------
+- Solvers: ``cfd_dem_coupling_3d``
+- Three-dimensional problem
+- Displays the selection of models and physical properties.
+- Simulates a solid-gas fluidized bed.
+
+
+Location of the examples
+------------------------
+ ``/examples/cfd-dem/square_fluidized_bed``
+
+
+Description of the case
+-----------------------
+
+This example simulates the fluidization of spherical particles in air. First, we use Lethe-DEM to fill the bed with particles. Then, we use the ``cfd_dem_coupling`` solver within Lethe to simulate the fluidization of the particles.
+
+
+Parameter file
+--------------
+
+The syntax is flexible. Parameters do not need to be specified in a specific order, but only within the subsection in which they belong. All parameter subsections are described in the `parameter section <../../../parameters.html>`_ of the documentation.
+
+To set-up the square fluidized bed case, we first fill the bed with particles. 
+We first introduce the different sections of the parameter file (packing_in_cylinder.prm) needed to run this simulation. In this example, we are simulating a squred fluidized that has a half length of 0.1 m, and a side of 0.04 m. We use the GridGenerator::subdivided_hyper_rectangle function of Deal.II in order to generate the mesh. The square bed is divided 40 times in the y direction. The following portion of the DEM parameter file shows the function called:
+
+
+Mesh
+~~~~~
+
+The ``mesh`` subsection specifies the computational grid:
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 				= dealii
+        set grid type            				= subdivided_hyper_rectangle
+        set grid arguments       				= 1,5,1:-0.02,-0.1,-0.02:0.02,0.1,0.02:true
+        set initial refinement   				= 3
+    end
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another subsection, which is generally the one we put at the top of the parameter files, is the ``simulation control`` . ``time step``, end time, log and ``output frequency`` are defined here. Additionally, users can specify the output folder for the simulation results in this subsection. The ``log frequency`` parameter controls the frequency at which the iteration number is printed on the terminal. If ``log frequency = 1000`` the iteration number will be printed out every 1000 iterations. This is an easy way to monitor the progress of the simulation. A simulation time of 1 s was chosen with a time step of 0.002. It it important to choose a long enough time as to allow all particles to come to rest. We store the output files generated in the folder output_dem:
```

Comment:
Wait this is the pure DEM simulation? I think the time-step is much lower than that in the pure DEM simulation no? 0.002s seems like a large time step for a DEM sim?

Reply:
Also if this is a pure DEM simulation, the bdf scheme and etc don't need to be specified right, because the lagrangian integrator is specified elsewhere.
Don't hesitate to remove parameters from the .prm file that are not useful.

Reply:
Done. I was confused with the cfd and dem parts of the example